server/routes/index.js

```diff
@@ -487,20 +488,6 @@ const routes = {
       locale = '/' + locale;
     }
 
-    let donation_url;
-
-    // don't fail if the donation_url isn't available,
-    // in case someone's browser isn't sending cookies
-    const encryptedCookie = request.state && request.state.session;
```

Comment:
Discovered on staging that browsers aren't forwarding the Cookie when paypal redirects to the callback. we'll need a different solution.